src/main/java/apoc/ApocConfiguration.java

```diff
@@ -27,7 +29,12 @@
 
     public static void initialize(GraphDatabaseAPI db) {
         Static.clear();
-        Map<String, String> params = db.getDependencyResolver().resolveDependency(Config.class).getRaw();
+        Config neo4jConfig = db.getDependencyResolver().resolveDependency(Config.class);
```

Comment:
this looks horribly complicated with little benefit.

Let's only grab the things we really need from the main config and that have the correct shape.